tokens.xml

```diff
@@ -3156,6 +3225,7 @@ Whenever Glyph Keeper becomes the target of a spell or ability for the first tim
             <reverse-related count="3">Siege-Gang Commander</reverse-related>
             <reverse-related count="2">Sling-Gang Lieutenant</reverse-related>
             <reverse-related>Survey the Wreckage</reverse-related>
+            <reverse-related count="2">You See a Pair of Goblins</reverse-related>
```

Comment:
Battle Cry Goblin creates 1/1 red goblins
Den of the Bugbear creates 1/1 red goblins

Reply:
Fixed.